test/e2e/set/nodeconfig/nodeconfig_disksetup.go

```diff
@@ -256,21 +257,21 @@ var _ = g.Describe("Node Setup", framework.Serial, func() {
 
 		defer func() {
 			framework.By("Stopping RAID device at %q", discoveredRaidDeviceOnHost)
-			stdout, stderr, err := executeInPod(f.KubeClient().CoreV1(), clientPod, "mdadm", "--stop", discoveredRaidDeviceOnHost)
+			stdout, stderr, err := executeInPod(f.AdminClientConfig(), f.KubeClient().CoreV1(), clientPod, "mdadm", "--stop", discoveredRaidDeviceOnHost)
 			o.Expect(err).NotTo(o.HaveOccurred(), stdout, stderr)
 		}()
 
 		framework.By("Checking if RAID device has been formatted")
 		o.Eventually(func(g o.Gomega) {
-			stdout, stderr, err := executeInPod(f.KubeClient().CoreV1(), clientPod, "blkid", "--output=value", "--match-tag=TYPE", discoveredRaidDeviceOnHost)
+			stdout, stderr, err := executeInPod(f.AdminClientConfig(), f.KubeClient().CoreV1(), clientPod, "blkid", "--output=value", "--match-tag=TYPE", discoveredRaidDeviceOnHost)
 			g.Expect(err).NotTo(o.HaveOccurred(), stderr)
 
 			g.Expect(strings.TrimSpace(stdout)).To(o.Equal("xfs"))
 		}).WithPolling(1 * time.Second).WithTimeout(3 * time.Minute).Should(o.Succeed())
 
 		framework.By("Checking if RAID was mounted at the provided location with correct options")
 		o.Eventually(func(g o.Gomega) {
-			stdout, stderr, err := executeInPod(f.KubeClient().CoreV1(), clientPod, "mount")
+			stdout, stderr, err := executeInPod(f.AdminClientConfig(), f.KubeClient().CoreV1(), clientPod, "mount")
```

Comment:
Isn't f.ClientConfig enough? It should have all necessary access in the namespace where this is executed.

Reply:
correct, done